website/pages/en/network/delegating.mdx

```diff
@@ -2,13 +2,23 @@
 title: Delegating
 ---
 
-Delegators are network participants who delegate (i.e., "stake") GRT to one or more Indexers. Delegators help secure the network without running a Graph Node themselves.
+Delegators are network participants who delegate (i.e., "stake") GRT to one or more Indexers.
 
-Delegators earn a portion of an Indexer's query fees and rewards by delegating to them. The amount of queries an Indexer can process depends on their own stake, the delegated stake, and the price the Indexer charges for each query. Therefore, the more stake that is allocated to an Indexer, the more potential queries they can process.
+- They help secure the network without running a Graph Node themselves.
+
+- They earn a portion of an Indexer's query fees and rewards by delegating to them.
+
+## How does this work?
+
+- The number of queries an Indexer can process depends on their own stake, **the delegated stake**, and the price the Indexer charges for each query. Therefore, the more stake allocated to an Indexer, the more potential queries an Indexer can process.
```

Comment:
Sorry if I'm too picky about this, but not sure why we ever have lists with a single item. Shouldn't this be a paragraph?

Reply:
No problem! Bullet points make it easier for UI

Reply:
-fine changing it for this 👍

Reply:
I'm curious to have @0xa3k5's thoughts on this.